src/vs/platform/terminal/common/terminal.ts

```diff
@@ -157,9 +291,9 @@ export interface IShellLaunchConfig {
 	extHostTerminalId?: string;
 
 	/**
-	 * This is a terminal that attaches to an already running remote terminal.
+	 * This is a terminal that attaches to an already running terminal.
 	 */
-	remoteAttach?: { id: number; pid: number; title: string; cwd: string; };
+	attachPersistentTerminal?: { id: number; pid: number; title: string; cwd: string; };
```

Comment:
Let's verify what pid, title and cwd are used for here. Could the `title` potentially get stale if it changed while the window was reloaded?

Reply:
how could the title change while reloading?

Reply:
On Mac or Linux the title is polled by calling `process` on node-pty, this is determined by what's going on in the shell but for example a command could finish during a reload and then this title would be stale. Maybe that's file and we just want to fill in the title faster. I'm wondering if we can avoid storing and passing this stuff back, that would also make things simpler.